### PR TITLE
Updates ‘/deploy’ to http://deployopcode.io/

### LINKED ISSUE #719: Create  Deploy Subdomain
As a VISITOR
When I VISIT DEPLOY
It should maybe BE A SUBDOMAIN instead of a standalone page so it won't get caught up in rails and would be easier to update, etc.  Just a thought to make life easier.  

Bret (and David)

### PR DESCRIPTION
**Pull request by feosuna1**
Closes #719. This PR routes all `/deploy` requests to http://deployopcode.io/.  I also removed the `deploy.html.erb` as it can no longer be accessed.

I'm interested in helping, this is my first commit.  Please let me know if I missed something.

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -19,7 +19,7 @@
           <li><a href="/mentorship">Software Mentorship</a></li>
           <li><a href="/scholarships">Scholarship Program</a></li>
           <li><a href="/employers">Employer Services</a></li>
-          <li><a href="/deploy">Web Application Development</a></li>
+          <li><a href="http://deployopcode.io/">Web Application Development</a></li>
```
Comment (rickr) [inline app/views/layouts/_footer.html.erb]: Since you've defined a route for deploy can we use that route here instead of hard coding the path?

Comment (feosuna1) [inline app/views/layouts/_footer.html.erb]: 👍 

I thought about.  I just didn't know if you minded the link saying `http://operationcode.org/deploy`.

**Comment by feosuna1** [comment]
@rickr Should I just revert all of the hard coded URL's back to `/deploy` since the updated route will perform a redirect?

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (rickr) [inline app/views/layouts/_footer.html.erb]: Ah - that's a good point. Probably better for SEO to link to it directly. Let's keep it like you have it.

**Pull request merged**